Wake On Lan/Modules/AboutScreen/Tests/AboutScreenTests/AboutScreenPresenterTests.swift

```diff
@@ -0,0 +1,68 @@
+import XCTest
+
+@testable import AboutScreen
+@testable import WOLResources
+
+final class AboutScreenPresenterTests: XCTest {
+    var sut: AboutScreenPresenter!
+    var interactorMock: AboutScreenInteractorInputMock!
+    var viewMock: AboutScreenViewInputMock!
+
+    override func setUp() {
+        super.setUp()
+        interactorMock = AboutScreenInteractorInputMock()
+        viewMock = AboutScreenViewInputMock()
+        sut = AboutScreenPresenter()
+        sut.interactor = interactorMock
+    }
+
+    func testViewDidLoad() {
+        // when
+        sut.viewDidLoad(viewMock)
+        // then
+        XCTAssertEqual(interactorMock.fetchBundleInfoCallsCount, 1, "Interactor must be called once")
+    }
+
+    func testIntercatorDidFetchBundleInfo() {
+        // given
+        let bundleInfo = TestData.bundleInfo
+        let viewModel = TestData.viewModel
+        // when
+        sut.interactor(interactorMock, didFetchBundleInfo: bundleInfo)
+        // then
+        XCTAssertEqual(viewMock.configureWithCallsCount, 1, "ViewController must be called once")
+        XCTAssertEqual(viewMock.configureWithReceivedViewModel?.buttonListViewModel.count,
+                       viewModel.buttonListViewModel.count,
+                       "the number of button items must equal 2"
```

Comment:
As far as I remember, there are 3 buttons

Reply:
There are three buttons. I made the correction. I also noticed that I inherited XCTest instead of XCTestCase - that was the reason I did not see this test failing.